vnext/Microsoft.ReactNative/Fabric/ComponentView.cpp

```diff
@@ -22,11 +22,12 @@ struct RootComponentView;
 
 namespace winrt::Microsoft::ReactNative::implementation {
 
-ComponentView::ComponentView(facebook::react::Tag tag, winrt::Microsoft::ReactNative::ReactContext const &reactContext)
-    : m_tag(tag), m_reactContext(reactContext) {}
-
-void ComponentView::MarkAsCustomComponent() noexcept {
-  m_customComponent = true;
+ComponentView::ComponentView(
+    facebook::react::Tag tag,
+    winrt::Microsoft::ReactNative::ReactContext const &reactContext,
+    winrt::Microsoft::ReactNative::Composition::ReactCompositionViewComponentBuilder *builder)
```

Comment:
`[understanding|non-blocking]` is the winrt pointer type auto-refcounted (like ObjC)?

Wonder a bit if passing a ref or r-value would make more sense for these delegates even if they point to an empty static no-op class of delegates

Reply:
ReactCompositionViewComponentBuilder is a winrt::implements struct, not a winrt interface or runtime type, so its not auto-refcounted.  Thats why the member variable is a com_ptr<ReactCompositionViewComponentBuilder> not just a ReactCompositionViewComponentBuilder.